.circleci/config.yml

```diff
@@ -12,6 +12,7 @@ jobs:
         environment:
           POSTGRES_USER: root
           POSTGRES_DB: circle_test
+          POSTGRES_PASSWORD: root_user_password
```

Comment:
this appeared to just make the build work, but I'm not 100% clear what this does. Is the password value actually used anywhere? I expected to have to pass one in through env variables in CircleCI or something